package.json

```diff
@@ -77,19 +78,11 @@
       "App": "app",
       "Component": "component",
       "StateClass": "state-class"
-    },
-    "mochaGlobals": [
-      "stub",
-      "spy",
-      "expect",
-      "_",
-      "Backbone",
-      "Marionette"
-    ]
+    }
   },
   "dependencies": {
```

Comment:
change this to `peerDependencies` and revert the changes on versions for `backbone` and `backbone.marionette` below